app/views/declaration/fiscalInformation/additional_fiscal_references.scala.html

```diff
@@ -76,11 +76,11 @@
                 Seq(
                     TableRow(
                         content = Text(additionalReference.countryName),
-                        attributes = Map("id" -> s"additional_fiscal_references-row$index-label")
+                        attributes = Map("id" -> s"additional_fiscal_references-row$index-country")
```

Comment:
Having `row$index` will result in sth like `row1`, `row2`. Are you sure this is what you want? Or maybe you'd prefer `row-1`, `row-2` ? I don't remember which way it is done in other similar places.

Reply:
I probably copied this from additional_information, but I can change it if the other way is better